### DISCUSSION TITLE
hubs view only spaces

### DISCUSSION THREAD

**Discussion by bharathreddyza** [mozilla/hubs#4546] [Hubs Cloud Questions] [answered]
is there any way to build a hubs space just for viewing purpose only.
A single user interacting in the space without any multi-user interaction and communication capabilities .
I haven't yet looked at the architecture of hubs  and hoping if there is already a way to do it

**Comment by pattersonbl2** [comment, marked as answer]
Ideally, you can do this with a hub cloud personal or a local install.  The hubs cloud would be an AWS marketplace set up.  It would have limited resources which would make it pretty affordable out of the box.  You can find more information here (https://hubs.mozilla.com/cloud)
